File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size() == 0) return 0;
        sort(nums.begin(), nums.end());
        vector<int> v;
        v.push_back(nums[0]);
        int ans = 0;
        for(int i=1; i<nums.size(); i++) {
            if(v[ans] != nums[i]) {
                v.push_back(nums[i]);
                ans++;
            }
        }
        for(int i=0; i<v.size(); i++) {
            nums[i] = v[i];
        }
        return ans + 1;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        ArrayList<Integer> u = new ArrayList<>();
        int ans = 0;
        u.add(nums[0]);
        for(int i=1; i<nums.length; i++) {
            if(u.get(ans) != nums[i]) {
                u.add(nums[i]);
                ans++;
            }
        }
        
        for(int i=0; i<ans + 1; i++) {
            nums[i] = u.get(i);
        }
        
        return ans + 1;
    }
}